pkg/plugin/sdk/deployment_source.go

```diff
@@ -128,7 +128,7 @@ func (c *ApplicationConfig[Spec]) parsePluginConfig(pluginName string) error {
 	}
 
 	// Validate the spec if it implements the Validate method.
-	if v, ok := any(spec).(interface{ Validate() error }); ok {
+	if v, ok := any(&spec).(interface{ Validate() error }); ok {
```

Comment:
Is this the same as just below?
`parsePluginConfig` checks both `Spec` and `*Spec` are implement `Validate() error`.

Reply:
It's a mistake. Thanks, fixed on [b5c2fa2](https://github.com/pipe-cd/pipecd/pull/5937/commits/b5c2fa2ee16cc8e496712f3df02fb323d7ec0add)